test/cri-containerd/runpodsandbox_test.go

```diff
@@ -1137,6 +1139,77 @@ func Test_RunPodSandbox_Mount_SandboxDir_LCOW(t *testing.T) {
 	//TODO: Parse the output of the exec command to make sure the uvm mount was successful
 }
 
+func Test_RunPodSandbox_CPUGroup(t *testing.T) {
+	ctx := context.Background()
+	presentID := "FA22A12C-36B3-486D-A3E9-BC526C2B450B"
+
+	processorTopology, err := uvm.HostProcessorInfo(ctx)
+	if err != nil {
+		t.Fatalf("failed to get host processor information: %s", err)
+	}
+	lpIndices := make([]uint32, processorTopology.LogicalProcessorCount)
+	for i, p := range processorTopology.LogicalProcessors {
+		lpIndices[i] = p.LpIndex
+	}
+
+	if err := cpugroup.CreateNewCPUGroupWithID(ctx, presentID, lpIndices); err != nil {
+		t.Fatalf("failed to create test cpugroup with: %v", err)
+	}
+
+	defer func() {
+		err := cpugroup.DeleteCPUGroup(ctx, presentID)
+		if err != nil && err != cpugroup.ErrHVStatusInvalidCPUGroupState {
+			t.Fatalf("failed to clean up test cpugroup with: %v", err)
+		}
+	}()
+
+	type config struct {
```

Comment:
(Doesn't have to be done in this PR but if you want 😉) Can we just move this as a non function local definition  seeming as how we use the same thing in 4 or so other tests? Obviously you'd just need to remove the declaration of this in the other tests also would be the only other work. 
```go	
type config struct {
		name             string
		requiredFeatures []string
		runtimeHandler   string
		sandboxImage     string
		containerImage   string
		cmd              []string
	}
```

Reply:
Yeah I was thinking about this too. Was planning to do it in a separate PR though.

Reply:
I'll do it real quick actually dw about it